zkevm-circuits/src/pi_circuit.rs

```diff
@@ -1632,6 +1894,9 @@ impl<F: Field> SubCircuit<F> for PiCircuit<F> {
             + 1 // for pi hash bytes start row
             + KECCAK_DIGEST_SIZE // pi hash bytes
             + 1 // for coinbase & difficulty start row
+            + 32 // for l1 block range hash
```

Comment:
```suggestion
            + KECCAK_DIGEST_SIZE // for l1 block range hash
```
And you are also missing the `max_l1_blockhashes * keccak_digest_size` here.